src/layout/Header/MobileMenu/MobileWeb3Status.tsx

```diff
@@ -36,7 +36,7 @@ const MobileWeb3Status: React.FC = () => {
               fontWeight={500}
               style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}
             >
-              {shortenAddress(account, chainId, 6)}
+              {account}
```

Comment:
i think first few digits and last few digits are important
![Screenshot 2022-07-06 at 3 48 31 PM](https://user-images.githubusercontent.com/6604146/177528568-5d257b4c-5b17-4754-af33-402475dd310a.png)